{
  try {
    Assert.assertNotNull(a);
  }
 catch (  AssertionError e) {
    assertionFailures.add(e);
  }
}
